spec/spec_helper.rb

```diff
@@ -4,7 +4,49 @@
 $:.unshift((ROOT + 'spec').to_s)
 
 require 'bundler/setup'
+require 'pry'
+
 require 'rspec'
 require 'danger'
 
 require 'danger_plugin'
+
+# These functions are a subset of https://github.com/danger/danger/blob/master/spec/spec_helper.rb
+# If you are expanding these files, see if it's already been done ^.
+
+# A silent version of the user interface,
+# it comes with an extra function `.string` which will
+# strip all ANSI colours from the string.
+
+# rubocop:disable Lint/NestedMethodDefinition
+def testing_ui
+  @output = StringIO.new
+  def @output.winsize
+    [20, 9999]
+  end
+
+  cork = Cork::Board.new(out: @output)
+  def cork.string
```

Comment:
Dunno if others will really have the problem of "I want to test the output" but y'know